core/src/test/scala/unit/kafka/utils/JaasTestUtils.scala

```diff
@@ -114,6 +114,9 @@ object JaasTestUtils {
     jaasFile.getCanonicalPath
   }
 
+  def clientLoginModule(mechanism: String, keytabLocation: Option[File]): String =
+    kafkaClientModule(mechanism, keytabLocation).toString.stripMargin
```

Comment:
Hmm, looks like we should add `stripMargin` to `JaasModule.toString` instead of here.